radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/MockedLedgerModule.java

```diff
@@ -30,34 +32,38 @@
 import com.radixdlt.consensus.epoch.EpochChange;
 import com.radixdlt.consensus.liveness.NextCommandGenerator;
 import com.radixdlt.consensus.sync.SyncRequestSender;
-import com.radixdlt.crypto.Hash;
+import com.radixdlt.universe.Universe;
 import com.radixdlt.consensus.PreparedCommand;
 import io.reactivex.rxjava3.core.Observable;
 
 public class MockedLedgerModule extends AbstractModule {
 	private final BFTValidatorSet validatorSet;
 
 	public MockedLedgerModule(BFTValidatorSet validatorSet) {
-		this.validatorSet = validatorSet;
+		this.validatorSet = Objects.requireNonNull(validatorSet);
 	}
 
 	@Override
 	public void configure() {
 		bind(CommittedStateSyncRx.class).toInstance(Observable::never);
 		bind(EpochChangeRx.class).toInstance(Observable::never);
-		EpochChange initialEpoch = new EpochChange(VertexMetadata.ofGenesisAncestor(validatorSet), validatorSet);
-		bind(EpochChange.class).toInstance(initialEpoch);
 		bind(NextCommandGenerator.class).toInstance((view, aids) -> null);
 		bind(SyncRequestSender.class).toInstance(req -> { });
 	}
 
+	@Provides
+	@Singleton
```

Comment:
Doesn't need to be singleton

Reply:
I think it does?  There are no guarantees here that the parameters are fixed.